zebra-state/Cargo.toml

```
@@ -19,11 +19,11 @@ serde = { version = "1", features = ["serde_derive"] }
 bincode = "1"
 
 futures = "0.3.17"
-metrics = "0.13.0-alpha.8"
-tower = { version = "0.4", features = ["buffer", "util"] }
+metrics = "0.17"
```

Let's use full versions, to avoid duplicate dependencies.

```suggestion
metrics = "0.17.0"
```

---

Done.